documentation/tutorials/creating_synthetic_tokens.md

```diff
@@ -0,0 +1,144 @@
+# Creating Synthetic Tokens from Truffle console
+
+This tutorial will show you how to create synthetic tokens from the command line using UMA’s synthetic token template. Before beginning this tutorial, please make sure your environment is set up correctly by following the instructions in the “Prerequisites” section. After completing this section, your environment will: 
+* Clone the UMA repo
+* Run Ganache on localhost port 9545
+* Compile all contracts in the UMA repo
+ 
+Below, we’ll discuss how to create and manage a token sponsor position via the command line on a local testnet. Please look here for information on existing UMA-supported testnet and mainnet deployments. 
+ 
+## Step 1: Deploy a contract and create tokens. 
+ 
+1. Set up the truffle environment and migrate contracts. (This step is not necessary if working with an existing testnet or mainnet deployment.)
+```
+$(npm bin)/truffle migrate --network test
+```
+ 
+2. Open the truffle console.
+```
+$(npm bin)/truffle console --network test
+```
+ 
+3. Create an instance of the expiring multiparty creator (the contract factory for synthetic tokens). This command should return “undefined”. 
+```
+const empCreator = await ExpiringMultiPartyCreator.deployed()
+```
+ 
+4. Define the parameters for the synthetic tokens you would like to create. For documentation surrounding the meanings of each parameter, please see this page. 
+ 
+```
+const constructorParams = { expirationTimestamp: "123456789000", withdrawalLiveness: "1000", siphonDelay: "100000", collateralAddress: TestnetERC20.address, tokenFactoryAddress: TokenFactory.address, priceFeedIdentifier: web3.utils.utf8ToHex("UMATEST"), syntheticName: "Test UMA Token", syntheticSymbol: "UMATEST", liquidationLiveness: "1000", collateralRequirement: { rawValue: web3.utils.toWei("1.5") }, disputeBondPct: { rawValue: web3.utils.toWei("0.1") }, sponsorDisputeRewardPct: { rawValue: web3.utils.toWei("0.1") }, disputerDisputeRewardPct: { rawValue: web3.utils.toWei("0.1") } }
```

Comment:
We no longer need to pass in `siphonDelay`, `tokenFactoryAddress`, `liquidationLiveness`, `withdrawalLiveness` and the `expirationTimestamp` must be one of these:

```
        1585699200, // 2020-04-01T00:00:00.000Z
        1588291200, // 2020-05-01T00:00:00.000Z
        1590969600, // 2020-06-01T00:00:00.000Z
        1593561600, // 2020-07-01T00:00:00.000Z
        1596240000, // 2020-08-01T00:00:00.000Z
        1598918400, // 2020-09-01T00:00:00.000Z
        1601510400, // 2020-10-01T00:00:00.000Z
        1604188800, // 2020-11-01T00:00:00.000Z
        1606780800, // 2020-12-01T00:00:00.000Z
        1609459200, // 2021-01-01T00:00:00.000Z
        1612137600, // 2021-02-01T00:00:00.000Z
        1614556800, // 2021-03-01T00:00:00.000Z
        1617235200, // 2021-04-01T00:00:00.000Z
        1619827200, // 2021-05-01T00:00:00.000Z
        1622505600, // 2021-06-01T00:00:00.000Z
        1625097600, // 2021-07-01T00:00:00.000Z
        1597017600 // 2020-08-10T00:00:00.000Z (Note: This is a special expiration that marks the end of the Tokyo Olympics 2020)
```

Reply:
Here is a suggested param object that fits these new restrictions: 
```
const constructorParams = { expirationTimestamp: "1585699200", collateralAddress: TestnetERC20.address, priceFeedIdentifier: web3.utils.utf8ToHex("UMATEST"), syntheticName: "Test UMA Token", syntheticSymbol: "UMATEST", collateralRequirement: { rawValue: web3.utils.toWei("1.5") }, disputeBondPct: { rawValue: web3.utils.toWei("0.1") }, sponsorDisputeRewardPct: { rawValue: web3.utils.toWei("0.1") }, disputerDisputeRewardPct: { rawValue: web3.utils.toWei("0.1") } }

```

I tested this locally as well tn double check :)